### DISCUSSION TITLE
Issues with getting student benefits

### DISCUSSION THREAD

**Discussion by SunderB** [community/community#21326] [GitHub Education] [answered]
Hi,
My student pack expired recently so I reapplied using my uni email and it was automatically accepted. However, 5 days later I don’t yet have access to any of the benefits even though I’ve been verified. In particular I need some GitHub Pro features for some projects that I’m working on - so I hope it can be resolved soon. 
Many thanks,
Sunder

Edit: I’ve noticed there’s a menu option to re verify your institution, but it takes me to the teacher application page!

**Comment by SunderB** [comment, marked as answer]
It seems like I’ve finally got access to student benefits. Thanks to whoever sorted it 🙂

**Comment by Miguel6** [comment]
Hi, I cannot use my student benefits (GitKraken Pro and Datadog Pro). Could someone help me?

**Comment by delbertaud** [comment]
I filled out the student signup about a week ago. I have yet to receive any confirmation. How do I know when, or if, I have been approved?

**Reply by FOREVEREALIZE** [reply]
You can see your aplication's status by going to [education.github.com/benefits](http://education.github.com/benefits)

It's at the right side

**Reply by delbertaud** [reply]
That link takes me to a page to sign up. No where there or on my profile does it have anything leading me to believe that I've been approved and I also have not received a confirmation email. Maybe because the school year is just starting they are backlogged with requests.

**Reply by FOREVEREALIZE** [reply]
> That link takes me to a page to sign up. No where there or on my profile does it have anything leading me to believe that I've been approved and I also have not received a confirmation email. Maybe because the school year is just starting they are backlogged with requests.

That would be weird because they processed my application in a couple of hours. 
If I go to the link above I see my applications. See here: [imgur](https://imgur.com/a/BTg8FV4)

You may see I'm at a different url but the one above just redirects here

**Reply by delbertaud** [reply]
hmmm, thank you for the information and image. Maybe I need to sign up again.

**Reply by delbertaud** [reply]
Thank you again. I did the process over and almost immediately got confirmed. I suspect my previous schedule wasn't excepted. Again BIG THANK YOU.

**Reply by Explorerho** [reply]
> Thank you again. I did the process over and almost immediately got confirmed. I suspect my previous schedule wasn't excepted. Again BIG THANK YOU.

What did you do differently this time please?

**Reply by delbertaud** [reply]
Nothing that I remember. I just started over again.

**Comment by JannemanDev** [comment]
I am a verified teacher and therefore I could use GitKraken Pro for free. But now after a year GitKraken gives an error "subscription has expired" and for example now I can't open all my private repos.
The webpage https://education.github.com/discount_requests/teacher_application says I dont need to re-verify: "You were last verified as a faculty on Sep 10, 2021. It is not necessary for you to re verify at this time."

How can I re-subscribe so I can use GitKraken Pro again?

**Reply by JannemanDev** [reply]
Never mind: the fix is to re-signin in GitKraken via File > Sign into a different account > Choose Github again. 
See also https://help.gitkraken.com/gitkraken-client/edu/

**Comment by Tobbyano1234** [comment]
Hi, I am having difficulty accessing my student pack after been verified. Please, I need help with this as I need some of this tool on some Project am working on currently. Thank is advance.

**Comment by danydaniel7** [comment]
I have applied for student and i got accepted. It has almost spent 3 days but i am not getting any student benefits.

**Comment by Kaifdoctor** [comment]
Whenever I am uploading a student card there is error I am facing please give me alternative solution
